### ISSUE TITLE
Uncaught TypeError: Cannot read property 'isElement' of null

### ISSUE THREAD

**Issue by alainlompo** [mulesoft/api-workbench#173]
[Enter steps to reproduce below:]

1. .Create a Collection and Member resource types as follow
resourceTypes:
  Collection:
    get:
      responses:
        200:
          body:
            application/json:
              type: <<item>>[]
    post:
      body:
        application/json:
          type: <<item>>
          example: |
            {
              "name":"Showly",
              "kind":"Carnivore",
              "price":100
            }
  Member:
    delete:
      responses:
        204:
    put:
      body:
        application/json:
          type: <<item>>
2. ...Use Contextual menu Refactoring => Extract Library
3.... Select the Collection and Member resource types for extraction
4.... Change the library name to CollectionsCoreLib and the file name to CollectionsCoreLib.raml
5. Click Extract

**Atom Version**: 1.7.3
**System**: Microsoft Windows 7 Edition Familiale Basique
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package, v0.8.28

### Stack Trace

Uncaught TypeError: Cannot read property 'isElement' of null

```
At C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:339

TypeError: Cannot read property 'isElement' of null
    at findDeclaration (C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:339:13)
    at getLibraryDefiningNodeByOffset (C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:929:23)
    at getLibraryDefiningNode (C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:922:22)
    at C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:863:46
    at Array.forEach (native)
    at refFinder (C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:817:18)
    at C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:812:9
    at Array.forEach (native)
    at refFinder (C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:811:21)
    at C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:812:9
    at Array.forEach (native)
    at Object.refFinder (C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\ast.core\search.js:811:21)
    at ASTNodeImpl.findReferences (C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\raml-1-parser\dist\raml1\highLevelImpl.js:923:16)
    at C:\Users\LOMPO\.atom\packages\api-workbench\main.js:5:6772
    at Array.filter (native)
    at C:\Users\LOMPO\.atom\packages\api-workbench\main.js:5:6737
    at Array.forEach (native)
    at e.performOk (C:\Users\LOMPO\.atom\packages\api-workbench\main.js:5:6709)
    at C:\Users\LOMPO\.atom\packages\api-workbench\main.js:5:2776
    at C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\atom-ui-lib\dist\UI.js:389:68
    at Array.forEach (native)
    at HTMLButtonElement.element.onclick (C:\Users\LOMPO\.atom\packages\api-workbench\node_modules\atom-ui-lib\dist\UI.js:389:31)
```

### Commands

```
     -4:35.6.0 api-workbench:Extract Library (atom-text-editor.editor.is-focused)
     -1:33.5.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.mini.is-focused)
  5x -1:32.8.0 core:move-left (atom-text-editor.editor.mini.is-focused)
 10x -1:30.9.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:48.1.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.mini.is-focused)
     -0:36.4.0 api-workbench:Extract Library (atom-text-editor.editor.is-focused)
     -0:21.2.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.mini.is-focused)
     -0:17 core:move-left (atom-text-editor.editor.mini.is-focused)
 10x -0:16.7.0 core:backspace (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
api-workbench, v0.8.28
linter, v1.11.4

# Dev
No dev packages
```

**Comment by alainlompo** [comment]
When the issue occurs and I cancel the extraction it generates an empty library file containing only the header: #%RAML 1.0 Library and no extraction

**Comment by ddenisenko** [comment]
Fixed. Fix published in 0.8.30.
